l10n_ch_account_statement_base_import/statement.py

```diff
@@ -71,8 +80,29 @@ def _statement_import(self, cr, uid, ids, prof, parser, file_stream,
         """
         self.file_stream = file_stream
 
-        return super(AccountStatementProfil, self)._statement_import(
-            cr, uid, ids, prof, parser, file_stream, ftype, context)
+        """
+        Calculate hash of the file to determinate if it already exist
+        """
+        hash = hashlib.md5(file_stream).hexdigest()
+        statement_obj = self.pool.get('account.bank.statement')
```

Comment:
no need to use `get()` you can write simply `self.pool['account.bank.statement']`